src/MCT.js

```diff
@@ -318,11 +316,17 @@ define([
      * @memberof module:openmct.MCT#
      * @method setAssetPath
      */
-    MCT.prototype.setAssetPath = function (path) {
-        this.legacyExtension('constants', {
-            key: "ASSETS_PATH",
-            value: path
-        });
+    MCT.prototype.setAssetPath = function (assetPath) {
+        this._assetPath = assetPath;
```

Comment:
Could we please conditionally add a trailing `/` if it's absent? This threw me when I tried to update the tutorial.